_commands/basics/mkdir.md

```diff
@@ -0,0 +1,98 @@
+---
+---
+
+mkdir
+--
+
+`mkdir` is used to create directories (if they do not already exist) on a file system.
+
+The general format is
+
+~~~ bash
+$ mkdir [options] directories
+~~~
+
```

Comment:
Please add a line after this one:

    <!--more-->

So ``/reference`` doesn't show your whole command.